src/tests/integration/id/profile/id-list-profile-dates.test.ts

```diff
@@ -0,0 +1,65 @@
+import { describe, expect, afterAll } from 'vitest'
+import { ID } from '../../../../id'
+import { testWithIDData } from '../test-with-id-data'
+import { ProfileDatesAttribute } from '../../../../types'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import randomString from '../../../lib/string/random'
+
+const CREATED_DATES_HANDLES : string[] = []
+const getHandle = () => { return `sdk_unit_test_dates_${randomString(5).toLowerCase()}_${Math.floor(Date.now() / 1000)}` }
+const testProfileID = process.env.OMNEO_TEST_PROFILE_ID as string
+
+describe('ID Dates', () => {
+  testWithIDData('ID SDK can get dates', async ({ IDData }) => {
+    const { tokenData } = IDData
+    const payload: ProfileDatesAttribute = {
+      name: 'Wedding',
```

Comment:
Could you please call this "SDK get dates"
In case one is left behind due to an error we can identify which test to fix

Reply:
changed.

Reply:
can you please change the `name` attribute on the payload. 
Apologies was not more specific